### ISSUE TITLE
SingleChildScrollView doesn't a11y announce its contents on initial focus

### ISSUE THREAD

**Issue by jklint-g** [flutter/flutter#33700]
Try this layout. When I launch the app on iOS with voiceover on, the scroll view is focused but nothing is read aloud.

```
@override
Widget build(BuildContext context) {
  return Center(child: SingleChildScrollView(child: Text('hello')));
}
```
Voiceover does read out "Possible text: hello" if you either:
- tap the scrollview
- swipe to navigate forward and then back

However I'd expect voiceover to read the contents on the initial focus when the screen appears. If I remove the scrollview, then it reads out 'hello' on launch.

This might be an iOS only issue, I haven't tried on android HW yet.

**Comment by dnfield** [comment]
Looks like we're failing to set the accessibilityLabel in this case.

**Comment by dnfield** [comment]
We weren't failing to set the label, we were allowing scrolling elements to gain focus on iOS when they shouldn't.  Fix in flight.

**Comment by dnfield** [comment]
(this won't be fixed until the engine rolls it in, which should be in the next 24 hours or less.

**Comment by dnfield** [comment]
Fixed as of https://github.com/flutter/flutter/pull/34310 - fa174a123fb603332e4870ccdd80bff9a1889f5b
